generator/main.go

```diff
@@ -65,10 +74,30 @@ func recordWritesErrored(count float64) {
 	writesErrored.Add(count)
 }
 
+func recordPatchesCompleted(count float64) {
+	patchesCompleted.Add(count)
+}
+
+func recordPatchesErrored(count float64) {
+	patchesErrored.Add(count)
+}
+
+const defaultRocksetEndpoint = "https://api.rs2.usw2.rockset.com"
```

Comment:
lets change this to `https://api.usw2a1.rockset.com`

Reply:
It isn't actually used after Martin's refactoring, so I just removed it